clients/src/main/java/org/apache/kafka/clients/consumer/internals/AbstractCoordinator.java

```diff
@@ -890,13 +890,20 @@ private synchronized Node coordinator() {
         return this.coordinator;
     }
 
-    protected synchronized void markCoordinatorUnknown() {
-        markCoordinatorUnknown(false);
+
+    protected synchronized void markCoordinatorUnknown(Errors error) {
+        markCoordinatorUnknown(false, "error response {}" + error.message());
```

Comment:
Wonder if we can use the name of the error. Some of the messages will read awkwardly in the log message

Reply:
Oops, that was what I was trying to do.

Reply:
Done